docs/source/assemblyguide.rst

```diff
@@ -0,0 +1,339 @@
+.. include:: global.rst.inc
+.. _assemblyguide:
+
+Assembly Guide
+==============
+
+.. Die erste Version dieser Anleitung stammte von einem Handout von Jürgen Böhringer mit dem Titel ``Wie baue ich einen Multigeiger?``
```

Comment:
If that document has some version / date, please add it here for reference, so we know precisely where this came from.

Reply:
will do